airflow/jobs/scheduler_job.py

```diff
@@ -1508,8 +1508,8 @@ def _execute_helper(self):
                               " have been processed {} times".format(self.num_runs))
                 break
 
-            if loop_duration < 1 and not is_unit_test:
-                sleep_length = 1 - loop_duration
+            if loop_duration < 0.5 and not is_unit_test:
```

Comment:
Can we make it configurable through `airflow.cfg`?

Reply:
yes, we can. Let me do that.